src/app.js

```diff
@@ -67,8 +67,8 @@ getManifest('manifest.webapp')
     .then(configI18n)
     .then(init)
     .then(
-        d2 => {
-            d2.system.settings
+        async d2 => {
```

Comment:
This is technically OK, but a bit weird to have a Promise `.then` within an `async/await` within another `.then`... why not either rewrite the whole function as in `async/await` style (cleaner) or keep the existing `.then` style with chained promises (smaller change)?

For the second option it would look something like this, since returning a promise in a `.then` callback will await that promise and await the result before calling the next `.then` callback: 

```js
.then(d2 =>
    d2.system.settings
        .get('keyBingMapsApiKey')
        .then(key => {
            store.dispatch(setBingMapsApiKey(key)))
            return d2 /* this could be avoided by using `getInstance()` below, but this is the existing behavior */
        })
).then(d2 => {
  const mapId = getUrlParameter('id')
  // ... etc ...
})
```

If you prefer the `async/await` addition as-is that's OK too, but in that case it should use the `await` rather than another `.then` inside the `async` function:

```js
    const key = await d2.system.settings.get('keyBingMapsApiKey')
    store.dispatch(setBingMapsApiKey(key))
```

Reply:
Thanks for review @amcgee! I agree that it's more readable - I've pushed a change that aligns with your first suggestion.